### DISCUSSION TITLE
Feature request: `--notebook-dir` support

### DISCUSSION THREAD

**Discussion by mawkler** [zk-org/zk-nvim#143] [Ideas]
### Verify

- [X] I searched the existing discussions for similar ideas

### Share your idea or feature request

I can't find any way to pass [`--notebook-dir`](https://github.com/mickael-menu/zk/blob/main/docs/notebook.md#notebook) to zk-nvim commands. Are there any plans to add support for this?

Example usage:

```lua
ZkNew { notebook_dir = "work" }
```

**Comment by mickael-menu** [comment]
Looking at the code, I think it's already supported as `notebook_path`.

**Reply by mawkler** [reply]
You're right! I missed it when searching through the docs. Thank you!
